azkaban-webserver/src/web/js/prettycron.js

```diff
@@ -0,0 +1,202 @@
+////////////////////////////////////////////////////////////////////////////////////
+//
+//  prettycron.js
+//  Generates human-readable sentences from a schedule string in cron format
+//
+//  Based on an earlier version by Pehr Johansson
+//  http://dsysadm.blogspot.com.au/2012/09/human-readable-cron-expressions-using.html
+//
+////////////////////////////////////////////////////////////////////////////////////
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published
+//  by the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+////////////////////////////////////////////////////////////////////////////////////
+
+if ((!moment || !later) && (typeof require !== 'undefined')) {
+  var moment = require('moment');
+  var later = require('later');
+}
+
+(function() {
```

Comment:
Open a ticket to update the moment.js so that we don't have to checkin this modified code?